pkgs/development/python-modules/online-judge-tools/api-client.nix

```diff
@@ -0,0 +1,41 @@
+{ appdirs, beautifulsoup4, buildPythonPackage, colorlog, fetchFromGitHub, git
+, jsonschema, lib, lxml, markdown, python, requests, substituteAll, toml }:
+
+let
+  pythonEnv =
+    python.withPackages (ps: with ps; [ colorlog jinja2 markdown toml ]);
+in buildPythonPackage rec {
+  pname = "online-judge-api-client";
+  version = "10.10.0";
+
+  src = fetchFromGitHub {
+    owner = "online-judge-tools";
+    repo = "api-client";
+    rev = "v${version}";
+    sha256 = "0lmryqi0bv82v9k9kf1rzzq9zr83smpmy8ivzw4fk31hvpczp4fn";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./api-client-fix-paths.patch;
+      inherit git;
+    })
+    (substituteAll {
+      src = ./api-client-fix-library-checker-problems-generate-py-deps.patch;
+      pythonInterpreter = pythonEnv.interpreter;
+    })
```

Comment:
This could also be done via substituteInPlace which maybe breaks less often. Either is fine.